row_081_u1.py

```
@@ -0,0 +1,499 @@
+        arguments = {"keywords": query, 
+                     "format": "jpg", 
+                     "limit":limit, 
+                     "print_urls":True,  
+                     "size": "medium"} 
+                       
+        # Providing arguments for the searched query 
+        try: 
+            # Downloading the photos based 
+            # on the given arguments 
+            response.download(arguments)  
+        except: 
```

Ruff (0.13.2)

Do not use bare `except`

(E722)